tests/plugins/pastefromwordimage/manual/pastewordwithimage.html

```diff
@@ -1,4 +1,7 @@
 <textarea id="editor" cols="30" rows="10"></textarea>
 <script>
+	if ( !CKEDITOR.plugins.clipboard.isCustomDataTypesSupported ) {
```

Comment:
This code should be called at least after [`pluginsLoaded`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.editor-event-pluginsLoaded) event as at this point it throws an error
![image](https://user-images.githubusercontent.com/1061942/31809684-08c1f33e-b579-11e7-81ae-6ed88d0d21f1.png)

because `CKEDITOR.plugins.clipboard` is not initialized yet.